ollama/_json_type_map.py

```diff
@@ -0,0 +1,111 @@
+import sys
+from typing import Any, List, Mapping, Optional, Sequence, Union, get_origin, get_args
+from collections.abc import Set
+from typing import Dict, Set as TypeSet, TypeVar
+
+T = TypeVar('T')
+if sys.version_info >= (3, 10):
+  from types import UnionType
+else:
+  UnionType = Union[T]
+
+# Python doesn't have a type serializer, so we need to map types to JSON types
+TYPE_MAP = {
```

Comment:
Need this ugly thing to make sure serialization is done correctly from type or str -> str